### DISCUSSION TITLE
IPv6 funktioniert nicht mehr

### DISCUSSION THREAD

**Discussion by vogler** [Freetz-NG/freetz-ng#629] [Allgemein]
Bei meiner FritzBox 7430 funktioniert seit einiger Zeit IPv6 nicht mehr richtig:
![image](https://user-images.githubusercontent.com/493741/194122109-aa4ac745-8f5c-4f07-b397-900647c06124.png)
Ich weiß nicht woran es liegen soll, es ging einfach von einen Tag auf den anderen nicht mehr. Vorher hatte ich bei https://ipv6-test.com/ einen Score von 17.
Es ist ein DSLite Anschluss, so komme ich jetzt von draußen gar nicht mehr in mein Heimnetz. Zudem funktionieren viele Apps und tools nicht mehr richtig (Disney+ auf TV/Android, Instagram auf Android, `npm`).

Ich habe eine zweite FritzBox 7430 ausprobiert. Nach Import aber das gleiche Problem. Nachdem ich die Settings ohne WLAN importiert habe, hatte IPv6 erstmal wieder funktioniert, dann nach einiger Zeit aber wieder nicht mehr. Es hat wohl etwas mit meinen Geräten im WLAN zu tun (die sich aber nicht geändert haben).
Bevor ich mir eine neue 7530 AX bestelle habe ich nun freetz-ng geflasht. Das hat erstmal nichts an der Situation geändert, aber evtl. kann mir damit jemand beim Debugging helfen. 🙏 
Ich denke der Zustand der Firewall ist irgendwie kaputt und lässt keine Pakete mehr durch, allerdings hat z.B. exposed host auch nicht geholfen.

```console
$ ping -c1 google.de
PING google.de (142.251.36.227): 56 data bytes
64 bytes from 142.251.36.227: icmp_seq=0 ttl=118 time=13.194 ms

1 packets transmitted, 1 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 13.194/13.194/13.194/0.000 ms

$ ping6 -c1 google.de
PING6(56=40+8+8 bytes) 2001:a61:35da:ec01:8d92:4f12:304c:69a1 --> 2a00:1450:4016:80a::2003
^C

1 packets transmitted, 0 packets received, 100.0% packet loss
```

![image](https://user-images.githubusercontent.com/493741/194123181-dfb4bec7-8021-40d0-97f7-a8cbd4071204.png)

**Comment by vogler** [comment]
Ok, nun habe ich eine 7530 AX gekauft. Neu eingerichtet, kein Import. Am Freitag hat alles inkl. IPv6 funktioniert, nun geht ping6 wieder nicht mehr :(
Irgendwie scheint die FritzBox Software allgemein mit etwas in meinem Netzwerk Probleme zu haben.

So sieht es z.B. auf meinem RPi aus:
```console
$ ping6 google.de
PING google.de(muc12s12-in-x03.1e100.net (2a00:1450:4016:809::2003)) 56 data bytes
64 bytes from muc12s12-in-x03.1e100.net (2a00:1450:4016:809::2003): icmp_seq=1 ttl=119 time=13.0 ms
64 bytes from muc12s12-in-x03.1e100.net (2a00:1450:4016:809::2003): icmp_seq=2 ttl=119 time=13.3 ms

# einige Zeit später

$ ping6 google.de
PING google.de(muc12s12-in-x03.1e100.net (2a00:1450:4016:809::2003)) 56 data bytes
^C
--- google.de ping statistics ---
6 packets transmitted, 0 received, 100% packet loss, time 5179ms
```

Settings:
```console
$ sudo sysctl net.ipv6.conf.all
net.ipv6.conf.all.accept_dad = 0
net.ipv6.conf.all.accept_ra = 1
net.ipv6.conf.all.accept_ra_defrtr = 1
net.ipv6.conf.all.accept_ra_from_local = 0
net.ipv6.conf.all.accept_ra_min_hop_limit = 1
net.ipv6.conf.all.accept_ra_mtu = 1
net.ipv6.conf.all.accept_ra_pinfo = 1
net.ipv6.conf.all.accept_ra_rt_info_max_plen = 0
net.ipv6.conf.all.accept_ra_rt_info_min_plen = 0
net.ipv6.conf.all.accept_ra_rtr_pref = 1
net.ipv6.conf.all.accept_redirects = 1
net.ipv6.conf.all.accept_source_route = 0
net.ipv6.conf.all.addr_gen_mode = 0
net.ipv6.conf.all.autoconf = 1
net.ipv6.conf.all.dad_transmits = 1
net.ipv6.conf.all.disable_ipv6 = 0
net.ipv6.conf.all.disable_policy = 0
net.ipv6.conf.all.drop_unicast_in_l2_multicast = 0
net.ipv6.conf.all.drop_unsolicited_na = 0
net.ipv6.conf.all.enhanced_dad = 1
net.ipv6.conf.all.force_mld_version = 0
net.ipv6.conf.all.force_tllao = 0
net.ipv6.conf.all.forwarding = 0
net.ipv6.conf.all.hop_limit = 64
net.ipv6.conf.all.ignore_routes_with_linkdown = 0
net.ipv6.conf.all.ioam6_enabled = 0
net.ipv6.conf.all.ioam6_id = 65535
net.ipv6.conf.all.ioam6_id_wide = 4294967295
net.ipv6.conf.all.keep_addr_on_down = 0
net.ipv6.conf.all.max_addresses = 16
net.ipv6.conf.all.max_desync_factor = 600
net.ipv6.conf.all.mc_forwarding = 0
net.ipv6.conf.all.mldv1_unsolicited_report_interval = 10000
net.ipv6.conf.all.mldv2_unsolicited_report_interval = 1000
net.ipv6.conf.all.mtu = 1280
net.ipv6.conf.all.ndisc_notify = 0
net.ipv6.conf.all.ndisc_tclass = 0
net.ipv6.conf.all.proxy_ndp = 0
net.ipv6.conf.all.ra_defrtr_metric = 1024
net.ipv6.conf.all.regen_max_retry = 3
net.ipv6.conf.all.router_probe_interval = 60
net.ipv6.conf.all.router_solicitation_delay = 1
net.ipv6.conf.all.router_solicitation_interval = 4
net.ipv6.conf.all.router_solicitation_max_interval = 3600
net.ipv6.conf.all.router_solicitations = -1
net.ipv6.conf.all.rpl_seg_enabled = 0
net.ipv6.conf.all.seg6_enabled = 0
net.ipv6.conf.all.suppress_frag_ndisc = 1
net.ipv6.conf.all.temp_prefered_lft = 86400
net.ipv6.conf.all.temp_valid_lft = 604800
net.ipv6.conf.all.use_oif_addrs_only = 0
net.ipv6.conf.all.use_tempaddr = 0

$ ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether b8:27:eb:73:c0:de brd ff:ff:ff:ff:ff:ff
    inet 192.168.178.3/24 brd 192.168.178.255 scope global dynamic eth0
       valid_lft 863212sec preferred_lft 755212sec
    inet6 2002:c0a8:b202:1:ba27:ebff:fe73:c0de/64 scope global dynamic mngtmpaddr noprefixroute
       valid_lft 86383sec preferred_lft 14383sec
    inet6 2001:a61:2b71:bc01:ba27:ebff:fe73:c0de/64 scope global dynamic mngtmpaddr noprefixroute
       valid_lft 6322sec preferred_lft 2722sec
    inet6 fe80::ba27:ebff:fe73:c0de/64 scope link
       valid_lft forever preferred_lft forever
```

**Comment by fda77** [comment]
Ich hab seit kurzem auch solche Probleme! Allerdings ist mein Router nicht von AVM
Kann es sein dass du auch bei dem Telekom bist? Ich glaub die haben da mal wieder was vergurkt. Ich bin es ja eigentlich gewohnt dass bei denen nichts richtig funktioniert... Ein Ticket öffne ich dort aber nicht. Ausser massenhaft Reconnects auslösen können die eh nicht

**Reply by vogler** [reply]
Ne, bin bei https://m-net.de
Ich hatte dort auch angerufen, dann hieß es nur von ihrer Seite aus sei alles in Ordnung. Bei AVM wollten sie mir gar nicht helfen weil der Support für meine 7430 schon ausgelaufen sei.
Wenn es tatsächlich am Provider liegt, kann ich natürlich lange Fehler suchen. Kann ich das irgendwie eingrenzen?
Was dagegen spricht ist das zeitweise ping6 von meinem rpi3 funktioniert hat und vom rpi4 nicht, dann einige Zeit später anders herum und jetzt von beiden nicht mehr.

**Comment by fda77** [comment]
Schade! Ich bin jedenfalls auch noch auf der Suche. https://www.ipv6-test.com/ hängt beim laden gern mal länger

![image](https://user-images.githubusercontent.com/471850/194926883-15887759-d178-4d00-b378-15caa0a9441f.png)

![image](https://user-images.githubusercontent.com/471850/194926896-0d12069f-2333-433e-ae87-6478e9c8640d.png)
